frontend/app/routes/$lang/_protected/applications/$id.tsx

```diff
@@ -55,21 +170,115 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   auditService.audit('page-view.application-details', { userId: idToken.sub });
   instrumentationService.countHttpStatus('application-details.view', 200);
 
-  return json({ viewApplication, submittedParsedFormat, meta });
+  return json({ applicationDetails, maritalStatus, mailingProvince, homeProvince, mailingCountry, homeCountry, communicationPreferenceLang, preferredLanguage, provincialTerritorialSocialProgram, federalSocialProgram, meta });
 }
 
 export default function ViewApplication() {
-  const { t } = useTranslation(handle.i18nNamespaces);
-  const { viewApplication, submittedParsedFormat } = useLoaderData<typeof loader>();
+  const { t, i18n } = useTranslation(handle.i18nNamespaces);
+  const { applicationDetails, maritalStatus, mailingProvince, homeProvince, mailingCountry, homeCountry, preferredLanguage, provincialTerritorialSocialProgram, federalSocialProgram, communicationPreferenceLang } = useLoaderData<typeof loader>();
+  const dateOfBirth = applicationDetails.dateOfBirth ? toLocaleDateString(parseDateString(applicationDetails.dateOfBirth), i18n.language) : '';
+
   const userOrigin = useUserOrigin();
   // TODO: Update with field mappings from Benefit Application from AB-3382
   return (
     <div className="max-w-prose">
-      <dl className="divide-y border-y">
-        <DescriptionListItem term={t('applications:view-application.submitted-date')}>{`${submittedParsedFormat}`}</DescriptionListItem>
-        <DescriptionListItem term={t('applications:view-application.status')}>{viewApplication.status}</DescriptionListItem>
-        <DescriptionListItem term={t('applications:view-application.confirmation-code')}>{viewApplication.confirmationCode}</DescriptionListItem>
-      </dl>
+      <div className="space-y-10">
+        <dl className="divide-y border-y">
+          <section className="space-y-6">
+            <h3 className="font-lato text-2xl font-bold">{t('applications:view-application.applicant-title')}</h3>
+            <dl className="divide-y border-y">
```

Comment:
We should validate this page against an HTML validator. There's some things I notice that would be caught such as `<section>` be a child of a `<dl>` and an `<h3>` after an `<h1>` and skipping an `<h2>`.

Reply:
Fixed: 4dddc042c12d4426462cf9e8f6eb625103dbf417